ci/asana-create-tasks.js

```diff
@@ -95,6 +96,22 @@ const waitForJobSuccess = async (job_gid) => {
     });
 };
 
+const getReleaseTasksSummary = async () => {
+    const { data: tasks } = await asana.tasks.getTasksForSection(scheduledForReleaseSectionGid, { opt_fields: 'name,permalink_url' });
```

Comment:
As I mentioned [in TD](https://app.asana.com/0/0/1209602007138039/1209772301776292/f), I _think_ it would mention tasks that were included in previous releases?

Reply:
Gah, I wrote a comment about this but it got stuck in an unpublished review 😅

<img width="834" alt="Screenshot 2025-03-25 at 10 05 05" src="https://github.com/user-attachments/assets/b171d439-8759-40e0-8fa2-f7927946ef3a" />

I like the idea you shared in the TD. Nice and simple. I’ll do that.